### PR TITLE
ticket 6 complete

### PR DESCRIPTION
**Pull request by omar-baggio**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -127,3 +127,22 @@ describe("GET /api/articles", () => {
...
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Array);
+        body.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("votes", expect.any(Number));
+          expect(comment).toHaveProperty("created_at", expect.any(String));
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("article_id", expect.any(Number));
+        });
```
Comment (Jodiech) [inline __tests__/app.test.js:148]: What happens when an article exists but has no comments? 🤔

File: __tests__/app.test.js [hunk 2]
```diff
@@ -127,3 +127,22 @@ describe("GET /api/articles", () => {
...
+      });
```
Comment (Jodiech) [inline __tests__/app.test.js:149]: There are a few tests missing here ❗ 
What if you are given an invalid article ID or a non-existent article ID?
Consider writing a test to check that the comments are sorted by created_at - you can use jest-sorted for this 👍

File: __tests__/app.test.js [hunk 3]
```diff
@@ -127,3 +127,22 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("status: 200, should responds with array of comments for a given article", () => {
+    return request(app)
+      .get(`/api/articles/1/comments`)
+      .expect(200)
+      .then(({ body }) => {
```
Comment (Jodiech) [inline __tests__/app.test.js]: I'd add a check for the length here to prevent any false positives if the array was empty!

File: endpoints.json [hunk 4]
```diff
@@ -42,6 +42,21 @@
...
+    "queries": [],
+    "exampleResponse": {
+      "article": {
+        "article_id": 3,
+        "body": "Ambidextrous marsupial",
+        "votes": 0,
+        "author": "icellusedkars",
+        "created_at": 1600560600000
+      }
+    }
```
Comment (Jodiech) [inline endpoints.json:57]: Nice to see you are updating your endpoints 😄

**Review by Jodiech — COMMENTED** [review]
Hey 👋  great work on this ticket. I've left a few comments for you below! Please make sure to include more tests for this ticket, for example - what happens when an article exists but there are no comments? You will need to change your model for this too. I'd like you to submit another PR for this please 🙏

**Review by lewispricey — APPROVED** [review]
Great work on this ticket Omar, thanks for implementing Jodie's feedback these extra tests are great.

Good to merge when you're ready! 🚀

**Pull request merged**